sim/battle.ts

```diff
@@ -1835,10 +1858,12 @@ export class Battle extends Dex.ModdedDex {
 
 		if (damage) {
 			if (this.gen <= 1 && effect.recoil && source) {
-				this.damage(this.clampIntRange(Math.floor(damage * effect.recoil[0] / effect.recoil[1]), 1), source, target, 'recoil');
+				this.damage(
+					this.clampIntRange(Math.floor(damage * effect.recoil[0] / effect.recoil[1]), 1), source, target, 'recoil');
```

Comment:
Generally I prefer to reduce line lengths by introducing temporary variables. Temporary variables are also good as documentation.

```js
				const amount = this.clampIntRange(Math.floor(damage * effect.recoil[0] / effect.recoil[1]), 1);
				this.damage(amount, source, target, 'recoil');
```